L1Trigger/Phase2L1ParticleFlow/python/l1pfJetMet_cff.py

```diff
@@ -1,10 +1,10 @@
 import FWCore.ParameterSet.Config as cms
 
-from L1Trigger.Phase2L1ParticleFlow.L1SeedConePFJetProducer_cfi import L1SeedConePFJetProducer, L1SeedConePFJetEmulatorProducer
-from L1Trigger.Phase2L1ParticleFlow.DeregionizerProducer_cfi import DeregionizerProducer as l1ctLayer2Deregionizer
-scPFL1PF            = L1SeedConePFJetProducer.clone(L1PFObjects = 'l1ctLayer1:PF')
-scPFL1Puppi         = L1SeedConePFJetProducer.clone()
-scPFL1PuppiEmulator = L1SeedConePFJetEmulatorProducer.clone(L1PFObject = cms.InputTag('l1ctLayer2Deregionizer:Puppi'))
+from L1Trigger.Phase2L1ParticleFlow.L1SeedConePFJetProducer_cfi import l1tSeedConePFJetProducer, l1tSeedConePFJetEmulatorProducer
+from L1Trigger.Phase2L1ParticleFlow.DeregionizerProducer_cfi import l1tDeregionizerProducer as l1tLayer2Deregionizer
+l1tSCPFL1PF            = l1tSeedConePFJetProducer.clone(L1PFObjects = 'l1tLayer1:PF')
+l1tSCPFL1Puppi         = l1tSeedConePFJetProducer.clone()
+l1tSCPFL1PuppiEmulator = l1tSeedConePFJetEmulatorProducer.clone(L1PFObject = cms.InputTag('l1tLayer2Deregionizer:Puppi'))
```

Comment:
```suggestion
l1tSCPFL1PuppiEmulator = l1tSeedConePFJetEmulatorProducer.clone(L1PFObject = 'l1tLayer2Deregionizer:Puppi')
```

Reply:
The suggested change fails with:
L1PFObject does not already exist, so it can only be set to a CMS python configuration type

Reply:
Well, it is to be spelled ```L1PFObjects``` rather than ```L1PFObject``` (which is exactly to king of bug to be caught with Andrea's suggestion)

Reply:
> The suggested change fails with: L1PFObject does not already exist, so it can only be set to a CMS python configuration type

Correct, `1tSeedConePFJetEmulatorProducer` is needed because some of its elements are cloned into the other one. Sorry for the wrong suggestion (and please also check the other ones, I did not test any of them...)

Reply:
Corrected